Sources/Fullscreen/LoadingView/LoadingView.swift

```diff
@@ -119,55 +127,93 @@ import UIKit
+        addSubview(successImageView)
         addSubview(loadingIndicator)
         addSubview(messageLabel)
-        addSubview(successImageView)
+
         NSLayoutConstraint.activate([
             successImageView.widthAnchor.constraint(equalToConstant: loadingIndicatorSize),
             successImageView.heightAnchor.constraint(equalToConstant: loadingIndicatorSize),
             successImageView.centerXAnchor.constraint(equalTo: centerXAnchor),
-            successImageView.centerYAnchor.constraint(equalTo: centerYAnchor, constant: -.mediumSpacing),
 
             loadingIndicator.widthAnchor.constraint(equalToConstant: loadingIndicatorSize),
             loadingIndicator.heightAnchor.constraint(equalToConstant: loadingIndicatorSize),
             loadingIndicator.centerXAnchor.constraint(equalTo: centerXAnchor),
-            loadingIndicator.centerYAnchor.constraint(equalTo: centerYAnchor, constant: -.mediumSpacing),
 
-            messageLabel.topAnchor.constraint(equalTo: loadingIndicator.bottomAnchor, constant: .mediumLargeSpacing),
-            messageLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: .veryLargeSpacing),
-            messageLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -.veryLargeSpacing)
+            messageLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: .smallSpacing),
+            messageLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -.smallSpacing),
+            messageLabel.topAnchor.constraint(equalTo: loadingIndicator.bottomAnchor, constant: .mediumSpacing)
         ])
     }
 
-    private func startAnimating(withMessage message: String? = nil) {
+    private func addHUD(displayType: DisplayType) {
         if superview == nil {
-            defaultWindow?.addSubview(self)
-            fillInSuperview()
+            guard let window = defaultWindow else { return }
+            window.addSubview(self)
+
+            switch displayType {
+            case .fullscreen:
+                successImageView.tintColor = .secondaryBlue
+                messageLabel.textColor = .licorice
+                backgroundColor = UIColor.white.withAlphaComponent(0.8)
+                layer.cornerRadius = 0
+                fillInSuperview()
+            case .boxed:
+                successImageView.tintColor = .milk
+                messageLabel.textColor = .milk
+                backgroundColor = UIColor.black.withAlphaComponent(0.8)
+                layer.cornerRadius = 16
+
+                NSLayoutConstraint.activate([
+                    heightAnchor.constraint(equalToConstant: boxedSize),
+                    widthAnchor.constraint(equalToConstant: boxedSize),
+                    centerXAnchor.constraint(equalTo: window.centerXAnchor),
+                    centerYAnchor.constraint(equalTo: window.centerYAnchor)
+                    ])            }
```

Comment:
A newline instead of spaces would be nice 😄

Reply:
Done!